scss/_variables.scss

```diff
@@ -299,17 +299,29 @@ $table-dark-color:           $body-bg !default;
 //
 // For each of Bootstrap's buttons, define text, background and border color.
 
-$input-btn-padding-y:       .375rem !default;
-$input-btn-padding-x:       .75rem !default;
-$input-btn-line-height:     $line-height-base !default;
+$input-padding-y:       .375rem !default;
+$input-padding-x:       .75rem !default;
+$input-line-height:     $line-height-base !default;
 
-$input-btn-padding-y-sm:    .25rem !default;
-$input-btn-padding-x-sm:    .5rem !default;
-$input-btn-line-height-sm:  $line-height-sm !default;
+$input-padding-y-sm:    .25rem !default;
+$input-padding-x-sm:    .5rem !default;
+$input-line-height-sm:  $line-height-sm !default;
 
-$input-btn-padding-y-lg:    .5rem !default;
-$input-btn-padding-x-lg:    1rem !default;
-$input-btn-line-height-lg:  $line-height-lg !default;
+$input-padding-y-lg:    .5rem !default;
+$input-padding-x-lg:    1rem !default;
+$input-line-height-lg:  $line-height-lg !default;
+
+$btn-padding-y:       $input-padding-y !default;
```

Comment:
Once we move input variables under Form we will be changing the order, so we will have to refence `btn-*` to `input-*` instead of the other way around.

@mdo we don't usually cross reference variables. Do you think we should add values absolute values in rem instead of abstracting them this way?

Reply:
@mdo I did so as not to break the old structure. No differences in the absolute values or copied from the variables. I think it is better to make the absolute values, it is also easier to change